src/index.html

```diff
@@ -1,17 +1,146 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <form method="post" action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+
+      <fieldset class="form-fieldset">
+        <legend>Personal information</legend>
+
+        <label 
+          class="form-field">
+          Surname:
+          <input 
+            type="text" 
+            id="surname"
+            name="surname" 
+            autocomplete="off" 
+            required
+          >
+        </label>
+        <label 
+          class="form-field">
+          Name:
+          <input type="text" 
+            id="name" 
+            name="name" 
+            minlength="3" 
+            maxlength="15" 
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field">
+        How old are You?
+          <input type="number" 
+          id="age" name="age" 
+          value="12" min="1" 
+          max="100"
+          >
```

Comment:
```suggestion
        <label class="form-field">
          How old are You?
          <input 
            type="number" 
            id="age"  
            name="age" 
            value="12" 
            min="1" 
            max="100"
           >
```
start each property on new line (CHECKLIST P3)  
fix it everywhere